### DISCUSSION TITLE
Virt-Manager Integration

### DISCUSSION THREAD

**Discussion by mrbrutti** [utmapp/UTM#4630] [Ideas]
It would be super useful to be able to manage other Qemu/ libvirt instances; much like Virt-Manager but within UTM. Virt-Manager and virt-viewer is not stable on macOS and UTM seams to be the natural evolution of such an integration in macOS.  

I would like to start a discussion on this idea and how it could benefit UTM as a product. For example VMware Fusion has a capability to connect to esxi servers and connect to those VMs running in other nodes. It would be tremendously useful for me and I assume others to have a similar experience on UTM.

**Comment by davidbitterlich** [comment]
Hi, this discussion is open for a year now and I've found it via google. I also would love to see such an integration. Virt-Manager can basically be used using Homebrew on macOS, but it's not integrated at all - and probably not as stable as it could be.

Libvirt is well documented and it would be a great extension for UTM to be able to manage those instances directly.

**Comment by irusensei** [comment]
Revisiting this now that we can run an UTM in server mode would it be a good addition if the frontend could also connect to libvirt? It would be nice to be able to connect on old KVM libvirt on amd64 machines.
